nwb_conversion_tools/utils/spike_interface.py

```diff
@@ -95,17 +95,15 @@ def get_nwb_metadata(recording: SpikeInterfaceRecording, metadata: dict = None):
     return metadata
 
 
-def add_devices(recording: SpikeInterfaceRecording, nwbfile=None, metadata: dict = None):
+def add_devices(nwbfile: pynwb.NWBFile, metadata: dict = None):
```

Comment:
so add_devices doesn't use the recording object?

Reply:
Nope lol, we've known that for a long time but kept the `recording` argument for posterity. Maybe the new ProbeInterface might have some kind of API value related to devices, but neither spikeextractors nor spikeinterface have explicit usage for that, so this is all dependent on `metadata` and default values